conf/defaults.ini

```diff
@@ -326,6 +326,7 @@ client_id = some_id
 client_secret = some_secret
 scopes = user:email
 email_attribute_name = email:primary
+email_attribute_jmes_path =
```

Comment:
I'd just call this `email_attribute_path` and remove the references to JMESPath all over, the fact that the path uses JMESPath syntax is a documentation item and doesn't need to be baked into all the names.